app/controllers/Application.scala

```diff
@@ -7,17 +7,22 @@ import services.IdentityService
 
 import scala.concurrent.ExecutionContext
 import cats.implicits._
+import wiring.ApplicationConfiguration
 
 class Application(
     actionRefiners: CustomActionBuilders,
     val assets: AssetsResolver,
     identityService: IdentityService,
     components: ControllerComponents
-)(implicit val ec: ExecutionContext) extends AbstractController(components) {
+)(implicit val ec: ExecutionContext) extends AbstractController(components) with ApplicationConfiguration {
```

Comment:
I would pass the bit of config you need into this class to avoid having some of the dependencies passed as values and some as mixed-in traits.